docs/source/changelog.rst

```diff
@@ -2,6 +2,60 @@
 Changelog
 =========
 
+Version 2022.10.0
+=================
+
+`Full changelog <https://github.com/jupyterhub/repo2docker/compare/2022.02.0...2022.10.0>`_
+
+New features
+------------
+
+- Update Jupyter dependencies :pr:`1193` by :user:`jtpio`
+- add Python 3.10 base environment :pr:`1175` by :user:`minrk`
+- Bump default R version to 4.2 from 4.1, and let R 3.4 go from 3.4.0 to 3.4.4 :pr:`1165` by :user:`yuvipanda`
+- Support pulling from zenodo sandbox too :pr:`1169` by :user:`yuvipanda`
+- Add MPDL Dataverse :pr:`1167` by :user:`wilhelmfrank`
+- Add JPL Dataverse :pr:`1163` by :user:`foobarbecue`
+- upgrade RStudio Server to v2022.02.1 :pr:`1148` by :user:`aplamada`
+- Pass build_args to `render()` during `--no-build` for consistency with regular builds :pr:`1135` by :user:`yoogottamk`
+
+Documentation improvements
+--------------------------
+
+- Update 'how to get R' section :pr:`1147` by :user:`yuvipanda`
+- Post release fixes :pr:`1133` by :user:`manics`
+
+API changes
+-----------
+
+Bug fixes
+---------
+- consistent log handling when not using JSON loggers :pr:`1177` by :user:`minrk`
+- Fix Read-Only filesystem permission issue for log file :pr:`1156` by :user:`timeu`
+- handle permission issue writing .jupyter-server-log.txt in REPO_DIR :pr:`1151` by :user:`pymonger`
+- handle r version being unspecified in environment.yml :pr:`1141` by :user:`minrk`
+
+Other merged PRs
+----------------
+
+- Remove conda buildpacks pin of r-irkernel to 1.2 :pr:`1191` by :user:`consideRatio`
```

Comment:
```suggestion
- Remove conda buildpacks pin of r-irkernel to 1.2 :pr:`1191` by :user:`consideRatio`
-  ci: refactor julia/r/conda tests - now ~25 min instead of ~50 min :pr:`1188` by :user:`consideRatio`
```